src/styles/card.scss

```diff
@@ -0,0 +1,21 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  &__name {
+    padding-top: 207px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $text-color-strong;
+    padding-inline: 16px;
+    transition: color $anim-duration ease;
```

Comment:
The variable `$anim-duration` is used but not defined. Please define this variable to avoid compilation errors.